src/main.ts

```diff
@@ -327,7 +327,8 @@ export class Extension {
         this.texMagician = new TeXMagician(this)
         this.envPair = new EnvPair(this)
         this.section = new Section(this)
-        this.latexCommandTreeView = new LaTeXCommandTreeView()
+        this.latexCommanderProvider = new LaTeXCommanderProvider(this)
```

Comment:
`class Extention` should not have providers. Remove `Extension.latexCommanderProvider`.